planning/behavior_path_planner/autoware_behavior_path_goal_planner_module/src/goal_planner_module.cpp

```diff
@@ -387,19 +393,59 @@ void LaneParkingPlanner::onTimer()
     /*forward_only_in_route*/ false);
   std::vector<PullOverPath> path_candidates{};
   std::optional<Pose> closest_start_pose{};
-  double min_start_arc_length = std::numeric_limits<double>::max();
+  std::optional<std::vector<size_t>> sorted_indices_opt{std::nullopt};
+  if (parameters_.bus_stop_area.use_bus_stop_area && switch_bezier_) {
+    bezier_planning_helper(
+      local_planner_data, goal_candidates, upstream_module_output, current_lanes,
+      closest_start_pose, path_candidates, sorted_indices_opt);
+  } else {
+    shift_planning_helper(
+      local_planner_data, goal_candidates, upstream_module_output, current_lanes,
+      closest_start_pose, path_candidates);
+  }
+
+  // set response
+  {
+    original_upstream_module_output_ = upstream_module_output;
+    std::lock_guard<std::mutex> guard(mutex_);
+    response_.pull_over_path_candidates = path_candidates;
+    if (closest_start_pose) {
+      response_.closest_start_pose = closest_start_pose;
+    }
+    RCLCPP_INFO(
+      getLogger(), "generated %lu pull over path candidates",
+      response_.pull_over_path_candidates.size());
+    response_.sorted_bezier_indices_opt = std::move(sorted_indices_opt);
+  }
+}
+
+void LaneParkingPlanner::shift_planning_helper(
```

Comment:
this helper is not only for shift. geometric_pull_over is also called in this helper

```
for (const auto & planner : pull_over_planners_) {
```

Reply:
fixed in bc8f304f9598ed381c9920c56d0da82fc82ac63a